pixart.js

```diff
@@ -1 +1,35 @@
+var colorField;
+var chosenColor;
 
+var setBoxColor = function(event){
+  event.preventDefault();  //if don't have this page is refreshed
+  colorField = document.getElementById("color-field")
+  chosenColor = colorField.value
+  var brush = document.querySelector(".brush")
+  brush.style.backgroundColor = chosenColor
+}
+
+//want backgorund color of box
+var setColorButton = document.getElementById("set-color")
```

Comment:
i generally like to keep all my var declarations/instantiations at the top of my page